pom.xml

```diff
@@ -81,8 +81,8 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <!-- Current plugin version -->
-        <fabric8.vertx.plugin.version>${project.version}</fabric8.vertx.plugin.version>
+        <!-- Current stable plugin version -->
+        <fabric8.vertx.plugin.version>1.0.0</fabric8.vertx.plugin.version>
```

Comment:
@rhuss dont use the property `fabric8.vertx.plugin.version` this property is used as part of the $project.basedir/src/main/resources/vertx-maven-plugin.properties to identify the current plugin version which is then used in various test cases. thats why I see the tests failing.  I will update this property to another property name and push it back